tests/ui/redundant_pattern_matching_option.rs

```diff
@@ -63,6 +63,8 @@ fn main() {
     } else {
         3
     };
+
+    issue7921();
```

Comment:
This shouldn't be necessary, since unused code is generally allowed in UI tests. 

EDIT: It may be necessary for the fixed test to pass. If so, just ignore my comment and resolve this thread.